src/index.ts

```diff
@@ -107,7 +107,14 @@ const getAssetFromKV = async (event: FetchEvent, options?: Partial<Options>): Pr
     pathIsEncoded = true;
     requestKey = request
   } else {
-    requestKey = options.mapRequestToAsset(request)
+    const mapedRequest = mapRequestToAsset(request)
```

Comment:
```suggestion
    const mappedRequest = mapRequestToAsset(request)
```

Reply:
Should be `mapped` instead of `maped` across the board here - can you update the rest of this code path too?